Src/Logic/LoginButton/LoginButton.ts

```diff
@@ -2,23 +2,26 @@ import {readFromLocalStorage} from '../LocalStorage/LocalStorage';
 import {User} from '../../../Models/User.model';
 import {setLoggedInUser, getLoggedInUser} from '../../UIComponents/utils/utils';
 import {USER_INPUT_EMPTY} from '../../../Constants/consts';
-function loginButton(username: string, userDashboard: (user: User) => HTMLElement, failComp: HTMLParagraphElement): void {
+import showModalWindow from '../../../Src/UIComponents/ModalWindow/ModalWindow';
+function loginButton(username: string, userDashboard: (user: User) => HTMLElement): void {
```

Comment:
wrong user type... instead User should be string
function loginButton(username: string, userDashboard: (user: **string**) => HTMLElement): void {

Reply:
No problem, I will changed that, just confirm me please, because that type (User) is used in master and I have not changed it.

Reply:
Done